assets/src/dashboard/utils/validateGoogleAnalyticsIdFormat.js

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-const googleAnalyticsIdFormatRegex = /^ua-\d+-\d+$/;
+const gaTrackingFormatOrGa4MeasurementId = /^ua-\d+-\d+|g-\d{7}$/;
```

Comment:
My apologies, but I think I didn't share the right link to the docs when I opened #5808.

Did some more digging and it seems that the format is way less strict.

It's difficult to see because the docs sometimes use `G-XXXXXXXXXX`, sometimes `G-XXXXXXX`. But apparently the measurement ID can contain up to 10 characters after the `G-` - letters and numbers. Example: `G-1A2BCD345E`

So I suggest using a less strict `g-[\w]+` regex for that.


```suggestion
const gaTrackingFormatOrGa4MeasurementId = /^ua-\d+-\d+|g-[\w]+$/;
```

Reply:
Don't apologize, this is super good info! I'll add some more tests for sanity checks.